internal/pkg/vm/address/address.go

```diff
@@ -101,6 +101,21 @@ func (a Address) Empty() bool {
 	return a == Undef
 }
 
+// MarshalBinary returns the address bytes
+func (a *Address) MarshalBinary() ([]byte, error) {
+	return a.Bytes(), nil
+}
+
+// UnmarshalBinary unmarshals the address bytes into an address
+func (a *Address) UnmarshalBinary(b []byte) error {
+	addr, err := NewFromBytes(b)
+	if err != nil {
+		return err
+	}
+	*a = addr
+	return err
+}
+
```

Comment:
This should have been replaced with the shared address package a long time ago.

Reply:
Agreed, and @acruikshank is on it!